docs/userguide/storagedriver/overlayfs-driver.md

```diff
@@ -0,0 +1,190 @@
+<!--[metadata]>
++++
+title = "OverlayFS storage in practice"
+description = "Learn how to optimize your use of OverlayFS driver."
+keywords = ["container, storage, driver, OverlayFS "]
+[menu.main]
+parent = "mn_storage_docker"
++++
+<![end-metadata]-->
+
+# Docker and OverlayFS in practice
```

Comment:
Is it still called OverlayFS? (I remember it being renamed to just 'overlay', but possibly that's just the package)